frappe/utils/xlsxutils.py

```diff
@@ -31,7 +31,7 @@ def get_excel_date_format():
 
 
 # return xlsx file object
-def make_xlsx(data, sheet_name, wb=None, column_widths=None):
```

Comment:
Can you add type hints and a docstring for the params/return type?

PR looks good otherwise.